packages/client/tests/e2e/default-version/package.json

```diff
@@ -9,5 +9,10 @@
   "devDependencies": {
     "@types/jest": "29.5.12",
     "@types/node": "18.19.50"
+  },
+  "pnpm": {
+    "onlyBuiltDependencies": [
+      "@prisma/client"
+    ]
```

Comment:
ℹ️ This is the actual fix for the failing test. `pnpm` v10 enforces that post install script need to be allowlisted. Our other tests passed because they don't rely on the script but perform a `prisma generate` explicitly. This test cannot do it as it doe snot install the prisma CLI itself. => We allowlist the script.